src/cf-modals/src/Modal.js

```diff
@@ -0,0 +1,57 @@
+/* ==========================================================================
+   Modal Organism
+   ========================================================================== */
+
+'use strict';
+
+const Organism = require( 'atomic-component/src/components/Organism' );
+
+const ModalOrganism = Organism.extend( {
+
+  insideClicked: false,
+
+  classes: {
+    isVisible: 'o-modal__visible'
+  },
+
+  events: {
+    'click .o-modal_close': 'onModalCloseClick',
+    'click .o-modal_backdrop': 'onModalCloseClick',
+    'click .o-modal_offset': 'onModalCloseClick',
+    'click .o-modal_inner': 'onModalPreventHide'
+  },
+
+  ui: {
+    base: '.o-modal'
+  },
+
+  onModalCloseClick: onModalCloseClick,
+  onModalPreventHide: onModalPreventHide
+} );
+
+/**
+ * Handle a click to close the modal.
+ * Applied to the modal close button, backdrop, and offset.
+ */
+function onModalCloseClick() {
+
+  // TODO: Fix line is temporary till AtomicComponent sets this.ui.base
+  //       to equal this.element.
+  this.ui.base = this.element;
```

Comment:
I would just move this to the initialize method, so it only gets called once when the component is instantiated.